cryptokit/Sources/CryptoKitSrc/hash.swift

```diff
@@ -68,3 +67,299 @@ public func SHA512(
     let hashData = hash.withUnsafeBytes { Data($0) }
     hashData.copyBytes(to: outputPointer, count: hashData.count)
 }
+
+@_cdecl("NewMD5")
+public func NewMD5() -> UnsafeMutableRawPointer {
+    let hasher = UnsafeMutablePointer<Insecure.MD5>.allocate(capacity: 1)
+    hasher.initialize(to: Insecure.MD5())
+    return UnsafeMutableRawPointer(hasher)
+}
+
+@_cdecl("MD5Write")
+public func MD5Write(_ ptr: UnsafeMutableRawPointer, _ data: UnsafePointer<UInt8>, _ length: Int) -> Int {
```

Comment:
This function is not mutating `ptr`, can we use `UnsafeRawPointer` instead?. Same for all other functions.